lib/components/chat_history/twitch/poll_event.dart

```diff
@@ -0,0 +1,159 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:provider/provider.dart';
+import 'package:rtchat/models/messages/twitch/event.dart';
+import 'package:rtchat/models/style.dart';
+import 'dart:math' as math;
+
+List<Widget> getPollsWidget(List<dynamic> choices, List<Widget> children,
```

Comment:
It would be better to wrap this as a `StatelessWidget` instead of returning a list of widgets.